virt/virtual_machines/virt-accessing-vm-consoles.adoc

```diff
@@ -26,6 +26,8 @@ include::modules/virt-vm-rdp-console-web.adoc[leveloffset=+2]
 [id="virt-accessing-vm-consoles-cli"]
 == Accessing virtual machine consoles by using CLI commands
 
+include::modules/virt-copying-the-ssh-command.adoc[leveloffset=+2]
```

Comment:
I think this module should be moved under the "Accessing virtual machine consoles in the OpenShift Container Platform web console" section.